docs/rules/fluent-chaining.md

```diff
@@ -0,0 +1,159 @@
+# Enforce formatting for chained calls (fluent-chaining)
+
+This rule enforces consistent spacing and line
+breaks when working with chained expressions.
+
+## Rule Details
+
+This rule aims to provide a consistent format
+when working with chained functions such as
+those provided by lodash or Q promises.
+
+## Examples
+
+Function body should be on separate lines.
+
+Invalid example:
+
+```js
+a()
+   .b(function() { return; });
+```
+
+Valid example:
+
+```js
+a()
+   .b(function() {
+      return;
+   });
+```
+
+Exception: Empty anonymous functions
+
+```js
+a(function() {});
+```
+
+---
+
+Member expressions shouldn't span more than 2 lines
+
+Invalid example:
+
+```js
+a()
+
+   .b();
+```
+
+Valid example:
+
+```js
+a()
+   .b();
+```
+
+---
+
+Once statment is broken into multiple lines 
+each member expression should be on it's own line
+
+Invalid example:
+
+```js
+a()
+   .b().c;
+```
+
+Valid example:
+
+```js
+a().b().c;
+
+// OR
+
+a()
+   .b()
+   .c;
+```
+
+---
+
+No spacing when member expression is on multiple lines
+
+Invalid example:
+
+```js
+a()
+.b();
+```
+
+Valid example:
+
+```js
+a()
+   .b();
+```
+
+---
+
+Function calls must begin and end with the same tab spacing
```

Comment:
Also, this "begin and end" at the same indentation level really doesn't have anything to do with "chaining" per se. I thought we had discussed making this a separate rule?